src/Tickets/Commerce/Status/Pending.php

```diff
@@ -165,6 +166,16 @@ public function can_apply_to( $order, $new_status ) {
 					$message = sprintf( __( 'There are no %s available at this time.', 'event-tickets' ), tribe_get_ticket_label_plural( 'unavailable_mixed' ) );
 				}
 
+				$tickets = tribe( Module::class )->get_tickets( $post->ID );
+
+				/**
+				 * Filters the unavailability message for a ticket collection
+				 *
+				 * @param string Unavailability message
+				 * @param array Collection of tickets
+				 */
+				$message = apply_filters( 'event_tickets_unvailable_message', $message, $tickets );
```

Comment:
Should we take all tickets here, or only the current one, given that the message applies only to one ticket at a time?

Reply:
Agreed. Should really only need to pass the `$ticket` and then you can remove the call to get the `$tickets` on line 169.

Reply:
Done. Please review.